backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewEdgeService.java

```diff
@@ -182,4 +200,63 @@ public boolean checkTransitionEdgeTarget(Element source, Element target) {
         }
         return false;
     }
+
+    /**
+     * Succession edge label getter service.
+     *
+     * @param succession
+     *            The succession edge
+     * @return the label of the given {@link Succession}
+     */
+    public String getSuccessionLabel(Succession succession) {
```

Comment:
I think this rather should be in `ViewLabelService`. What do you think about?

Reply:
Agree. Moved.